Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -99,6 +100,31 @@ def homepage(homepage = nil)
     @homepage ||= homepage
   end
 
+  def language(*args, &block)
+    @language ||= {}
+
+    if !args.empty? && block_given?
+      args.each do |arg|
+        MacOS.languages.each_with_index do |l, index|
+          string_or_regex = arg == :default ? %r{^en} : arg
+          next unless l.match(string_or_regex)
+          next unless @language[:level].nil? || @language[:level] > index
+          @language = {
+                        block: block,
```

Comment:
This is the style according to the current RuboCop config for Cask, so I'm leaving this as-is for now as it is used throughout `cask/*`.